examples/model_selection/plot_cv_indices.py

```diff
@@ -12,10 +12,15 @@
 
 """
 
+import warnings
+
 import matplotlib.pyplot as plt
 import numpy as np
 from matplotlib.patches import Patch
 
+# Removing warnings from examples
+warnings.filterwarnings("ignore")
```

Comment:
I think this removes all warnings. I would prefer that we only remove UserWarnings.